db.json

```diff
@@ -0,0 +1,59 @@
+{
+  "indicator": [
+    {
+      "id": 1,
+      "name": "Indicator1",
+      "query": "",
+      "slug": "indicator-1",
+      "category": "wildlife",
+      "threshold": 1,
```

Comment:
I've set an object in threshold attr in case the query returns a number or other string different than the one we are going to work with